dcat/rdf/dcat3.ttl

```diff
@@ -481,7 +581,31 @@ dcat:Role
   skos:scopeNote "Anvendes i forbindelse med kvalificerede krediteringer til at angive aktørens rolle i forhold til en entitet. Det anbefales at værdierne styres som et kontrolleret udfaldsrum med aktørroller, såsom http://registry.it.csiro.au/def/isotc211/CI_RoleCode."@da ;
   skos:scopeNote "Anvendes i forbindelse med kvalificerede relationer til at specificere en entitets rolle i forhold til en anden entitet. Det anbefales at værdierne styres med et kontrolleret udfaldsrum for for entitetsroller såsom: ISO 19115 DS_AssociationTypeCode http://registry.it.csiro.au/def/isotc211/DS_AssociationTypeCode; IANA Registry of Link Relations https://www.iana.org/assignments/link-relation;  DataCite metadata schema;  MARC relators https://id.loc.gov/vocabulary/relators."@da ;
 .
+
+dcat:previousVersion
+  a rdf:Property ;
+  a owl:ObjectProperty ; 
+  rdfs:label "previous version"@en ;
+  rdfs:label "versión anterior"@es ;
+  rdfs:label "versione precedente"@it ;
+  rdfs:comment "The previous version of a resource in a lineage [PAV]."@en ;
+  rdfs:comment "La versione precedente di una risorsa."@it ;
+  skos:definition "The previous version of a resource in a lineage [PAV]."@en ;
+  skos:definition "La versione precedente di una risorsa."@it ;
+  skos:scopeNote "This property is meant to be used to specify a version chain, consisting of snapshots of a resource."@en ;
+  skos:scopeNote "Questa proprietà è usata per specificare una catena di versioni, costituita da snapshot di una risorsa."@it ;
+  skos:scopeNote "The notion of version used by this property is limited to versions resulting from revisions occurring to a resource as part of its life-cycle. One of the typical cases here is representing the history of the versions of a dataset that have been released over time."@en ;
+  skos:scopeNote "La nozione di versione usata da questa proprietà è limitata a versioni risultanti da revisioni a cui una risorsa è soggetta nel suo ciclo di vita. Uno dei casi tipici è la rappresentazione della storia delle versioni di un dataset, che sono state pubblicate nel corso del tempo."@it ;
+  skos:changeNote "New property added in DCAT 3."@en ;
+  skos:changeNote "Nueva propiedad agregada in DCAT 3."@es ;
+  skos:changeNote "Nová vlastnost přidaná ve verzi DCAT 3."@cs ;
+  skos:changeNote "Nuova proprietà aggiunta in DCAT 3."@it ;
+  skos:changeNote "Ny egenskab tilføjet i DCAT 3."@da ;
+.
+
+
```

Comment:
```suggestion
```suggestion
dcat:hasVersion
  a rdf:Property ;
  a owl:ObjectProperty ; 
  owl:equivalentProperty pav:hasVersion ;
  rdfs:subPropertyOf dcterms:hasVersion ;
  rdfs:subPropertyOf prov:generalizationOf ;
  rdfs:label "has version"@en ;
  rdfs:label "TBD"@es ;
  rdfs:label "ha versione"@it ;
  rdfs:comment "This resource has a more specific, versioned resource [PAV]."@en ;
  rdfs:comment "TBD"@es ;
  rdfs:comment "Per questa risorsa esiste una risorsa più specifica e versionata."@it ;
  skos:definition "This resource has a more specific, versioned resource [PAV]."@en ;
  skos:definition "TBD"@es ;
  skos:definition "Per questa risorsa esiste una risorsa più specifica e versionata."@it ;
  skos:scopeNote "This property is intended for relating a non-versioned or abstract resource to several versioned resources, e.g., snapshots [PAV]."@en ;
  skos:scopeNote "TBD"@es ;
  skos:scopeNote "Questa proprietà è usata per correlare una risorsa non versionata o astratta a differenti risorse versionate, ad es., i relativi snapshot."@it ;
  skos:scopeNote "The notion of version used by this property is limited to versions resulting from revisions occurring to a resource as part of its life-cycle. Therefore, its semantics is more specific than its super-property dcterms:hasVersion, which makes use of a broader notion of version, including editions and adaptations."@en ;
  skos:scopeNote "TBD"@es ;
  skos:scopeNote "La nozione di versione usata da questa proprietà è limitata a versioni risultanti da revisioni a cui una risorsa è soggetta nel suo ciclo di vita. Quindi la sua semantica è più specifica di quella della sua super-proprietà dcterms:hasVersion, che utilizza una nozione di versione più ampia, e include, ad es., edizioni e adattamenti."@it ;
  skos:changeNote "New property added in DCAT 3."@en ;
  skos:changeNote "Nueva propiedad agregada in DCAT 3."@es ;
  skos:changeNote "Nová vlastnost přidaná ve verzi DCAT 3."@cs ;
  skos:changeNote "Nuova proprietà aggiunta in DCAT 3."@it ;
  skos:changeNote "Ny egenskab tilføjet i DCAT 3."@da ;
.

dcat:hasCurrentVersion
  a rdf:Property ;
  a owl:ObjectProperty ; 
  owl:equivalentProperty pav:hasCurrentVersion ;
  rdfs:subPropertyOf pav:hasVersion ;
  rdfs:label "has current version"@en ;
  rdfs:label "TBD"@es ;
  rdfs:label "ha versione attuale"@it ;
  rdfs:comment "This resource has a more specific, versioned resource with equivalent content [PAV]."@en ;
  rdfs:comment "TBD"@es ;
  rdfs:comment "Per questa risorsa esiste una risorsa più specifica e versionata, ma con lo stesso contenuto."@it ;
  skos:definition "This resource has a more specific, versioned resource with equivalent content [PAV]."@en ;
  skos:definition "TBD"@es ;
  skos:definition "Per questa risorsa esiste una risorsa più specifica e versionata, ma con lo stesso contenuto."@it ;
  skos:scopeNote "This property is intended for relating a non-versioned or abstract resource to a single snapshot that can be used as a permalink to indicate the current version of the content [PAV]."@en ;
  skos:scopeNote "TBD"@es ;
  skos:scopeNote "Questa proprietà è usata per correlare una risorsa non versionata o astratta a un suo specifico snapshot che può essere usato come permalink per indicare la versione attuale del suo contenuto."@it ;
  skos:scopeNote "The notion of version used by this property is limited to versions resulting from revisions occurring to a resource as part of its life-cycle."@en ;
  skos:scopeNote "TBD"@es ;
  skos:scopeNote "La nozione di versione usata da questa proprietà è limitata a versioni risultanti da revisioni a cui una risorsa è soggetta nel suo ciclo di vita."@it ;
  skos:changeNote "New property added in DCAT 3."@en ;
  skos:changeNote "Nueva propiedad agregada in DCAT 3."@es ;
  skos:changeNote "Nová vlastnost přidaná ve verzi DCAT 3."@cs ;
  skos:changeNote "Nuova proprietà aggiunta in DCAT 3."@it ;
  skos:changeNote "Ny egenskab tilføjet i DCAT 3."@da ;
.

dcat:version
  a rdf:Property ;
  a owl:DatatypeProperty ; 
  owl:equivalentProperty pav:version ;
  rdfs:label "version"@en ;
  rdfs:label "TBD"@es ;
  rdfs:label "versione"@it ;
  rdfs:comment "The version indicator (name or identifier) of a resource."@en ;
  rdfs:comment "TBD"@es ;
  rdfs:comment "L'indicatore di versione (un nome o un identificatore) di una risorsa."@it ;
  skos:definition "The version indicator (name or identifier) of a resource."@en ;
  skos:definition "TBD"@es ;
  skos:definition "L'indicatore di versione (un nome o un identificatore) di una risorsa"@it ;
  skos:scopeNote "DCAT does not prescribe how a version name / identifier should be specified, and refers for guidance to [DWBP]'s Best Practice 7: Provide a version indicator."@en ;
  skos:scopeNote "TBD"@es ;
  skos:scopeNote "DCAT non prescrive come un nome o identificatore di versione dovrebbe essere specification, and fa riferimento alle linee guida indicate in [DWBP] Best Practice 7: Provide a version indicator."@it ;
  skos:changeNote "New property added in DCAT 3."@en ;
  skos:changeNote "Nueva propiedad agregada in DCAT 3."@es ;
  skos:changeNote "Nová vlastnost přidaná ve verzi DCAT 3."@cs ;
  skos:changeNote "Nuova proprietà aggiunta in DCAT 3."@it ;
  skos:changeNote "Ny egenskab tilføjet i DCAT 3."@da ;
.

```
```